production/loki-mixin-compiled-ssd/dashboards/loki-operational.json

```diff
@@ -2539,7 +2539,7 @@
                   "steppedLine": false,
                   "targets": [
                      {
-                        "expr": "topk(10,sum by (tenant) (loki_ingester_memory_streams{cluster=\"$cluster\",job=~\"($namespace)/(loki.*|enterprise-logs)-write\"}))",
```

Comment:
These selectors have changed a lot. Do the old job patterns not work or are we querying slightly different data here?